go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/kubevirt/kubevirt-tekton-tasks
 
 go 1.21
-toolchain go1.22.2
+toolchain go1.22.3
```

Comment:
Because the previous PR was closed, not to miss it:

@ksimon1 Can we update the project to `go 1.22`?